src/components/app.jsx

```diff
@@ -1,11 +1,12 @@
 import React, { Component } from 'react';
 import QnA from './QnA';
+import RnR from './RnR/RnR_container';
 import '../styles/standard-styles.scss';
 
 class App extends Component {
   constructor(props) {
     super(props);
-    this.state = this.props;
+    this.state = {};
```

Comment:
Looks good, I'll work on setting up initial stage state to get state for this component sorted out.